package.json

```diff
@@ -36,10 +36,12 @@
     "babel-preset-stage-1": "^6.5.0",
     "babel-register": "^6.11.6",
     "babel-runtime": "^6.11.6",
+    "bootstrap": "^3.3.7",
```

Comment:
OK, I understand the need to import the libraries but the above snippet needs to be refactored as well. 

1) We can't rely on code being loaded from a public CDN, that needs to be vendored into the application itself
2) ATM we have version information for jquery and bootstrap in two places which can easily get out of sync. This needs to be only in `package.json`.